code/style.css

```diff
@@ -1,3 +1,164 @@
+html,
 body {
-  background: #FFECE9;
-}
\ No newline at end of file
+  max-width: 100%;
+  overflow-x: hidden;
+}
+
+body {
+  background: #f7dd74;
```

Comment:
I really like the palette! The background color provides for good readability in contrast to the font. In terms of [website engagement](https://backcourtmarketing.com/%EF%BB%BFcan-colors-increase-website-engagement/), yellow and orange are positive colors that can invite the user to stay longer on your website.

Reply:
That's a great article! Thanks for the tip.